style.css

```diff
@@ -613,3 +621,229 @@ footer .btn:active {
   color: white;
   background: #2230d2;
 }
+
+@media screen and (min-width: 768px) {
+  #toolbar {
+    display: flex;
+    justify-content: space-around;
+    align-items: space-around;
+  }
+
+  .burger-menu {
+    display: none;
+    height: 0;
+    width: 0;
+    margin: 0;
+    padding: 0;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  nav ul {
+    width: 60%;
+    display: flex;
+    justify-content: flex-end;
+    gap: 24px;
+  }
+
+  nav ul li {
+    text-decoration: none;
+    list-style-type: none;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    color: #344563;
+  }
+
+  .logo-nav {
+    margin: 0 0 0 120px;
+  }
+
+  .hero {
+    align-items: center;
+    justify-content: center;
+    background-image: url(./desktopPics/Hero\ Desktop\ Background.svg);
+    background-size: 100% auto;
+    height: 916px;
+  }
+
+  .hero h1 {
+    width: 70%;
+    font-size: 48px;
+    color: #091e42;
+    font-weight: 700;
+  }
+
+  footer p {
+    width: 569px;
+    height: 102px;
+  }
+
+  .hero p {
+    width: 70%;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    color: #344563;
+  }
+
+  .hero h6 {
+    margin-right: 380px;
+  }
+
+  .social-container {
+    margin-right: 326px;
+  }
+
+  #work-article-1 {
+    order: 3;
+  }
+
+  #work-article-2 {
+    order: 4;
+  }
+
+  #work-article-3 {
+    order: 1;
+  }
+
+  #work-article-4 {
+    order: 2;
+  }
+
+  .card {
+    grid-gap: 4%;
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .card h2 {
+    font-size: 40px;
+  }
+
+  .w3imgmb {
+    display: none;
+  }
+
+  .w3imgdt {
+    display: block;
+  }
+
+  .snapshot {
+    flex: 1;
+  }
+
+  .leftblock {
+    flex: 1;
+  }
+
+  .leftblock2 {
+    flex: 1;
+  }
+
+  #swap-pos {
+    order: 2;
+    flex: 1;
+  }
+
+  #pos-swap {
+    order: 1;
+  }
+
+  .about-container {
+    display: flex;
+  }
+
+  .about {
+    margin: 100px 0;
+    width: 46%;
+    padding: 0 100px;
+  }
+
+  .about h2 {
+    margin: 26px 0 0 0;
+  }
+
+  section .languages {
+    width: 54%;
+    align-items: flex-start;
+  }
+
+  .lang-divs {
+    width: 600px;
+  }
+
+  .lang-skills {
+    width: 600px;
```

Comment:
- [ ] I wouldn't suggest using fixed widths since they can break site responsibility. Check these screenshots to see what I mean and kindly try to fix it: 
![image](https://user-images.githubusercontent.com/91486791/194408460-1dec3845-9330-4403-987e-75442d5634f8.png)
In a red background you can see the empty space on the right: 
![image](https://user-images.githubusercontent.com/91486791/194408930-ed8b5b3c-1e6f-44e0-879c-ac55bda5a7dc.png)

Reply:
Yes, you are absolutely right. I have changed fixed width values to percentages. Now it will not have any issues.🙃